src/main/java/io/github/pmckeown/dependencytrack/CommonConfig.java

```diff
@@ -14,18 +14,20 @@ public class CommonConfig {
     private String projectVersion;
     private String dependencyTrackBaseUrl;
     private String apiKey;
+    private boolean validateCertificationPath;
```

Comment:
This name isn't what I would choose.  Please rename to `verifySsl`.  Might as well keep it in line with the UniRest configuration it will ultimately set.

Also, it needs to default to `true`. 
```suggestion
    private boolean validateCertificationPath = true;
```

Reply:
I renamed the variable, however the default shouldn't be set here, it should be set where maven parameters are initialized: AbstractDependencyTrackMojo. Also I noticed that when calling the constructor of BomClient, the commonConfig object was not fully ready to be able to call Unirest.config().verifySsl(), so I moved that call in the methods right before the rest calls.